kuma/static/js/libs/mozilla.cookiehelper.js

```diff
@@ -41,15 +41,15 @@ Mozilla.Cookies = {
         var sExpires = '';
         if (vEnd) {
             switch (vEnd.constructor) {
-            case Number:
-                sExpires = vEnd === Infinity ? '; expires=Fri, 31 Dec 9999 23:59:59 GMT' : '; max-age=' + vEnd;
-                break;
-            case String:
-                sExpires = '; expires=' + vEnd;
-                break;
-            case Date:
-                sExpires = '; expires=' + vEnd.toUTCString();
-                break;
+                case Number:
```

Comment:
This should be submitted as a PR against https://github.com/mozilla/trafficcop as well.

Reply:
Good point, will do so.